Toad-Labs/vc-frontend-static
# Vcon 65 buttons

### [Feedback Button](https://decdvirtualconcierge.atlassian.net/browse/VCON-65?atlOrigin=eyJpIjoiNGVmNmZlN2M0NmFlNDRlOTg3YzZjZWRjZmI4MzFhOWYiLCJwIjoiaiJ9)

## Description 

Added a responsive button for reporting any feedback.

## How to test the PR

1. Checkout this branch (vcon-65-buttons)
2. npm run dev
3. change screen width to see how it responds

## Things to consider

This button is not in the same container as the conversation window, I'm not 100% sure if that's fine, but it should be next to the footer as it is right now regardless.
